pydream/Dream.py

```diff
@@ -55,11 +55,18 @@ class Dream():
         A model name to be used as a prefix when saving history and crossover value files.
     hardboundaries : bool
         Whether to relect point back into bounds of hard prior (i.e., if using a uniform prior, reflect points outside of boundaries back in, so you don't waste time looking at points with logpdf = -inf).
+    mp_context : a multiprocessing context or None. It will be used to launch the workers
```

Comment:
This should be in the format `mp_context: <type>` with the description below. Also would be helpful to add a comment about what None does, and when this option should be changed.